gathering/async/get.js

```diff
@@ -56,23 +56,38 @@ function reduceUpdates (thread) {
 }
 
 function reduceAttendees (myKey, thread) {
-  const attendees = thread
+  const { attendees, notAttendees } = thread
```

Comment:
woop! now i know what you meant by "should be an object" and not an array! Looks much better :D